addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd

```diff
@@ -63,6 +65,20 @@ func run(command_params: Array) -> int:
 
 # Function called when the command is interrupted.
 func interrupt():
-	escoria.object_manager.get_object(_snd_player).node.set_state(
-		previous_snd_state
-	)
+	var _sound_players = []
+	if previous_snd_state.empty():
+		previous_snd_state = "off"
+	
+	if _snd_player.empty():
+		_sound_players = [
+			ESCObjectManager.MUSIC, 
+			ESCObjectManager.SOUND, 
+			ESCObjectManager.SPEECH
+		]
+	else:
+		_sound_players = [_snd_player]
+	
+	for snd_player in _sound_players:
+		escoria.object_manager.get_object(snd_player).node.set_state(
```

Comment:
Similar question here re `node` being null. If you're not that worried about such a case happening, then that's ok.